BMM.Core/ViewModels/PlayerBaseViewModel.cs

```diff
@@ -28,9 +32,13 @@ public ITrackModel CurrentTrack
             get => _currentTrack;
             private set
             {
-                SetProperty(ref _currentTrack, value);
-                RaisePropertyChanged(() => Downloaded); // Since Downloaded depends on CurrentTrack we need to raise PropertyChanged
-                RaisePropertyChanged(() => IsSeekingDisabled);
+                _currentTrackDebounceDispatcher.Run(() =>
```

Comment:
can you please explain in a comment why you added the debouncer

Reply:
When changing the song on Android, exo player sends few current track updates, where few of them is just null. 

Example seuence looks like that:
Current Song => (next button clicked) => null => Current Song => null => null => Next Song

This is changing in very short time, so adding debuncer with small slippage prevents plaver view from flickering, as finally we react for only one update.